crates/ratchet-models/src/registry.rs

```diff
@@ -75,6 +76,7 @@ impl AvailableModels {
                 PhiVariants::Phi2 => "FL33TW00D-HF/phi2",
                 PhiVariants::Phi3 => "FL33TW00D-HF/phi3",
             },
+            AvailableModels::Moondream => "tgestson/ratchet-moondream2",
```

Comment:
Do we want to move the weights to a repo owned by you (or the project) to be more uniform?  They are now just created with candles tensor-tools from [this moondream2 model.safetensors file](https://huggingface.co/vikhyatk/moondream2/blob/2024-05-08/model.safetensors)

Reply:
@timgestson let's create a repo for the project!

Reply:
Moved to the community repo